src/editortoolkit_neume.cpp

```diff
@@ -1019,12 +1017,17 @@ bool EditorToolkitNeume::Merge(std::vector<std::string> elementIds)
         m_infoObject.import("message", "At least two staves must be provided.");
         return false;
     }
-
-    uly /= staves.size();
-    lry /= staves.size();
     StaffSort staffSort;
     std::sort(staves.begin(), staves.end(), staffSort);
```

Comment:
It may be more appropriate to use `sortByUlx` here if you're trying to figure out absolute upper left and lower right, but I'm not sure.